docs/qe-available-tasks/sealights/sealights-get-refs.md

```diff
@@ -0,0 +1,59 @@
+# Sealights Get Refs Tekton Task
```

Comment:
Can we add dependencies to this task?

Like what tasks need to happening before we are able to run this?

Reply:
Yes. Good point. I will add some examples

Reply:
done